### PR TITLE
Lighten blockquote colour in dark mode

### PR DESCRIPTION
**Pull request by resynth1943**
Fixes https://github.com/vector-im/element-web/issues/5154

image

image

Signed-off-by: Resynth <resynth1943@tutanota.com>

### CODE REVIEW

File: res/themes/dark/css/_dark.scss [hunk 1]
```diff
@@ -272,6 +272,10 @@ $composer-shadow-color: rgba(0, 0, 0, 0.28);
             background-color: #080808;
         }
     }
+
+    blockquote {
+        
+    }
```
Comment (t3chguy) [inline res/themes/dark/css/_dark.scss:278]: shouldn't there be something in here?
I'm surprised the linter hasn't harassed you about an empty block

Comment (resynth1943) [reply res/themes/dark/css/_dark.scss:278]: Gottem 😎

**Comment by t3chguy** [comment]
Err, now this PR has exactly -0+0 changes?

**Comment by resynth1943** [comment]
...wait, what?

**Review by turt2live — APPROVED** [review]
code seems fine

**Review by nadonomy — APPROVED** [review]

**Pull request merged**